docs/docs/services/qix-engine/access-control.md

```diff
@@ -141,78 +157,83 @@ In addition to this `user` will contain all other attributes defined in the
 
-A _Resource_ is a generic concept that can represent applications, or objects within applications. A Resource carries
-attributes which may be used in rule expressions. In rule expressions the resource being accessed is represented by the
-`resource` object.
+A resource object is a generic concept that can represent applications, or objects within applications.
+A resource object consists of attributes that can be used in rule expressions.
+In rule expressions, the resource being accessed is represented by the `resource` object.
 
-#### Common Attributes
+#### Common resource attributes
 
-There are different resource types but all share some common attributes:
+There are different resource types, but all resources share some common attributes.
 
 | Attribute | Description |
 | --------- | ----------- |
 | `resource.description` | The resource description. Can be empty. | `resource.description = "My custom description"` |
 | `resource.id` | The unique identifier for the resource. |
 | `resource.owner` | The owner of the resource. |
 | `resource._resourcetype` | The type of the resource being accessed. Equal to `"App"` or `"App.Object"`. |
-| `resource._actions` | The actions to be granted on the resource. See [Actions](#actions) for more details.|
+| `resource._actions` | The actions granted on the resource. See [Actions](#actions) for more details.|
 
-#### Type-specific Attributes
+#### Type-specific resource attributes
 
-Depending on what type of resource that is being accessed, `resource` may carry additional attributes. If the resource
-type is `"App"`, there are no additional attributes. If the resource type is `"App.Object"`, `resource` has the
-following additional attributes:
+Depending on the type of resource that is being accessed, `resource` may carry additional attributes.
+If the resource type is `"App"`, there are no additional attributes.
+If the resource type is `"App.Object"`, then `resource` has the following additional attributes:
 
 | Attribute | Description | Example condition |
 | --------- | ----------- | ----------------- |
 | `resource._objecttype` | The object type. | `resource._objecttype = "field" or resource._objecttype = "my-generic-object"` |
 | `resource.app` | Reference to the app resource that the object is part of. | `resource.app.owner = "john-doe"` |
 
-##### The `app` Attribute
+##### The `app` attribute
 
-As stated above, if `resource._resourcetype = "App.Object"`, `resource.app` contains the reference to the `"App"`
-resource that the object exists in, and all attributes and [built-in functions](#built-in_function) that apply
-to `"App"` objects are available on `resource.app`.
+When the common resource attribute is `resource._resourcetype = "App.Object"`,
+then the type-specific resource attribute `resource.app` contains a reference to the `app` resource.
+This means that all attributes and built-in functions are available on `resource.app`.
 
-#### Built-in Functions
+#### Built-in functions
```

Comment:
If omitting the **HasPrivilege** subsectin (since there is only one) I think it makes sense to rename the **Built-in functins** section to somthing like **`HasPrivilege` built-in function**.

Reply:
yes, I as hoping you'd comment here because I couldn't think of a good way to fix this section.